LICENCE.cloudpickle

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2012-now, CloudPickle developers and contributors.
```

Comment:
I'm not sure how we've done this in the past, but I think it probably makes sense to colocate the license with the source. That will also ensure it actually gets packed into the Python container.

Reply:
Actually, I think we need to just add it to https://github.com/apache/beam/blob/master/LICENSE directly

Reply:
I checked what is included in containers images:

./opt/apache/beam/third_party_licenses/cloudpickle/LICENSE  -- this would probably disappear if cloudpickle is not a dependency, unless we modify sdks/python/container/license_scripts/manual_licenses to copy it, which may not be necessary if we add the license in one of the two below places:

./opt/apache/beam/LICENSE
./opt/apache/beam/LICENSE.cloudpickle

I don't have preference which file to add it to, we can see what is simpler and easier to import internally.

Reply:
> Does LICENSE.cloudpickle work for building containers?

yes, it works, we can check that it is included in the container build. for example, by running

```
gradlew :sdks:python:container:py310:docker
```

and inspecting the image content.

Reply:
feel free to follow up in another PR if this needs another change.